test/testenv/appframework_utils.go

```diff
@@ -34,6 +35,9 @@ var RestartNeededApps = []string{"Splunk_TA_nix", "splunk_app_microsoft_exchange
 //NewAppsAddedBetweenPolls Apps to be installed as poll after
 var NewAppsAddedBetweenPolls = []string{"TA-LDAP"}
 
+//ESApp is just ES app as it requires special configuration
+var ESApp = []string{"SplunkEnterpriseSecuritySuite"}
```

Comment:
Create a list to store a single value seems a bit redundant. Can the app name be just stored inside the test itself ?